bg_atlasgen/mesh_utils.py

```diff
@@ -180,20 +181,22 @@ def create_region_mesh(args):
     ROOT_ID: int, id of root structure (mesh creation is a bit more refined for that)
     """
     # Split arguments
-    (
-        meshes_dir_path,
-        node,
-        tree,
-        labels,
-        annotated_volume,
-        ROOT_ID,
-        closing_n_iters,
-    ) = args
+    logger.debug(f"Creating mesh for region {args[1].identifier}")
+    meshes_dir_path = args[0]
```

Comment:
What is the rational for this parsing of a single args tuple? I can't understand how it would be better than position arguments definition.

Also, could we have defaults for no smoothing and no decimation?

Reply:
Maybe it's just me not being good at multi processing, but with `pool.map` I always pass a tuple of arguments otherwise I can't get it to work (this is for doing mesh creation with `PARALLEL=True`)

Reply:
Unfortunately I still can't test this today, but could this be of any help?

https://stackoverflow.com/questions/5442910/how-to-use-multiprocessing-pool-map-with-multiple-arguments

Is seems to be supported from Python 3.3 onward

Reply:
Cool, I can try.